Library/Formula/sdl2_mixer.rb

```diff
@@ -20,17 +20,30 @@ class Sdl2Mixer < Formula
   depends_on "flac" => :optional
   depends_on "fluid-synth" => :optional
   depends_on "smpeg2" => :optional
+  depends_on "libmodplug" => :optional
   depends_on "libmikmod" => :optional
   depends_on "libvorbis" => :optional
 
   def install
+    if build.with?("libmodplug") && !build.head? then
+      # libmodplug support was fixed in changeset 695 (https://hg.libsdl.org/SDL_mixer/rev/6a5e6d8d6a35).
+      odie "sdl2_mixer 2.0.0 doesn't properly support a compilation with libmodplug. Use --HEAD."
+    end
```

Comment:
Option is used by stable version. Moreover, it's enabled by default. But it's broken. I suppose, that libmodplug headers were moved since sdl2_mixer 2.0.0, so it cannot find them. 
Instead of checking `build.head?`, I can try to provide a patch for ver. 2.0.0 inside formula. 
Of course, removing this option is also possible. I just don't know, when next version of sdl2_mixer will be made. Version 2.0.0 was made in 2013.08. Fix for libmodplug support was in 2014.08. There are bunch of other fixes in upstream. Seems like developers are just not marking them as new versions.